README.md

```diff
@@ -2,4 +2,31 @@
 
 [![Build Status](https://secure.travis-ci.org/twitter/finatra-activator-thrift-seed.png?branch=master)](http://travis-ci.org/twitter/finatra-activator-thrift-seed?branch=master)
 
-A minimal [Activator](https://www.lightbend.com/activator/docs) seed template for creating a Finatra Thrift server application.
\ No newline at end of file
+A minimal [Activator](https://www.lightbend.com/activator/docs) seed template for creating a Finatra Thrift server application.
+
+## Quick Start
+
+A simple client implementation
+
+```scala
+val client: PingService[Future] = ThriftMux.client
+    .configured(param.Tracer(NullTracer))
```

Comment:
Yeah , this is the old diff , fixed up over here https://github.com/twitter/finatra-activator-thrift-seed/pull/2/commits/98f38ce2e7a45fdca767f9d483873ed8517f8bda , will update that anonymous function as recommended though.